requirements.txt

```diff
@@ -1,3 +1,4 @@
 Markdown
 Pygments
-bottle
\ No newline at end of file
+bottle
+pymdown-extensions
```

Comment:
I'm sorry, it was the wrong link, according to https://facelessuser.github.io/pymdown-extensions/ it is ok ! sorry for the confusion...